src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -4,10 +4,34 @@
  * Add some custom fields that could be only in Bulldozer
  * Do not remove no-args constructor
  */
-public class Bulldozer extends Machine {
+public class Bulldozer extends Machine implements Workable {
```

Comment:
```suggestion
public class Bulldozer extends Machine {
```

Reply:
It's too much. Machine already implements this interface, so all inheritors will also implement this interface.